src/components/FormComponent/FormComponent.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import InputFieldComponent from '../InputFieldComponent/InputFieldComponent'
+
+class FormComponent extends Component {
+	constructor() {
+		super();
+		this.state = {
+			checkbox: false,
+			email: '',
+			firstName: '',
+			lastName: '',
+			password: '',
+		}	
+	}
+
+	handleInput = (e) => {
+		const target = e.target;
+	    const value = target.type === 'checkbox' ? target.checked : target.value;
+	    const name = target.name;
+
+	    console.log("Name: " + name);
+
+	    this.setState({
+	    	[name]: value
+	    });
+
+
+	}
+
+	handleSubmit = (e)  => {
+		e.preventDefault();
+		console.log("Name is: " + this.state.firstName);
+		console.log("Last Name is: " + this.state.lastName);
+		console.log("Password is: " + this.state.password);
+		console.log("Checkbox is: " + this.state.checkbox);
+	}
+
+	render () {
+		return (
+			<div>
+				<form onSubmit={this.handleSubmit}>
+					<InputFieldComponent 
+							name="firstName"
+							value={ this.state.firstName } 
+							type="text"
+							label="First name"
+							handleChange={ this.handleInput } />
+					<InputFieldComponent 
+							name="lastName"
+							value={this.state.lastName} 
+							type="text"
+							label="Last name"
+							handleChange={ this.handleInput } />
```

Comment:
Be consistent with spaces around curly braces {}, the styleguide you should follow is that there are no spaces around them when using JSX https://github.com/airbnb/javascript/tree/master/react